Hometask_1/Hometask_1.py

```diff
@@ -0,0 +1,37 @@
+import random  # importing random module
+#  getting 100 numbers from 0 to 1000
+hundred_random_numbers = []  # create a list
+for _ in range(100):  # define count of numbers in the list
+    hundred_random_numbers.append(random.randint(0, 1000))  # create the list of numbers from 0 to 1000
+# sort list from min to max
+sorted_numbers = []  # create a new list for further sorted numbers from min to max
+while hundred_random_numbers:
+    minimum = hundred_random_numbers[0]  # put min value on the beginning of the list
+    for a in hundred_random_numbers:
```

Comment:
Please, use 'number' instead of 'a'.
You should never (almost never) use variable names like 'i', 'x', 'a' in your project.